lib/KalturaHelpers.php

```diff
@@ -195,6 +239,18 @@ public static function getOption( $name, $default = null ) {
 			return $default;
 		}
 	}
+	
+	public static function getPlayerDimension($default = '16:9' ) {
```

Comment:
Need to refactor this method to have the default `16:9` be returned from the method and not externally.